### PR TITLE
add mapped LeadiD request variables to response data (fixes #8)

### LINKED ISSUE #8: include the input leadid values in the appended data
Per discussion with Alex for a customer with complex logic to determine the value mapped to LeadiD Lead Provider Code ("lpc"), we'd like to "echo" those input values back in the response. That way they can be used subsequently without repeating that complex logic.

### PR DESCRIPTION
**Pull request by cgrayson**
includes minor refactoring of tests now that mapped inputs are included in default test data

### CODE REVIEW

File: src/audit.coffee [hunk 1]
```diff
@@ -114,6 +118,13 @@ response.variables = ->
   [
     { name: 'outcome', type: 'string', description: 'Integration outcome', examples: ['success', 'failure', 'error'] }
     { name: 'reason', type: 'string', description: 'If outcome is error, the error reason', examples: ['duplicate'] }
+
+    # append the mapped request data in the response
+    { name: 'account_code', type: 'string', description: 'LeadiD Account Code ("lac")' }
+    { name: 'agency_code', type: 'string', description: 'LeadiD Agency Code ("agency")' }
+    { name: 'audit_key', type: 'string', description: 'LeadiD Audit Key ("lak")' }
```
Comment (alexkwolfe) [inline src/audit.coffee]: I'm wondering about these three keys. Do we need to include them? They're LeadiD GUIDs, aren't they? If so, I can't think of a reason we'd need them later in the flow.

Comment (cgrayson) [inline src/audit.coffee]: Yeah, that's true. I made the assumption that if one input were to be echoed, all of them should be. But maybe they'd  be mapped via rules, too?

Comment (cgrayson) [inline src/audit.coffee]: Though to your point, they're LeadiD-specific. Seems unlikely they'd be needed for anything other than this audit call itself. I'll remove them.

Comment (alexkwolfe) [inline src/audit.coffee]: They might be mapped via rules in the LeadiD step. But unless you're making more than one LeadiD call in the same flow, I can't think of a reason you'd need them later on. The values only seem useful for integrating with LeadiD.

Comment (alexkwolfe) [inline src/audit.coffee]: We crossed streams. Message received.

Comment (cgrayson) [inline src/audit.coffee]: We crossed streams. Message received.

**Comment by alexkwolfe** [comment]
LGTM

**Pull request merged**